src/resources/postcss/tribe-events-single-blocks-reskin.pcss

```diff
@@ -0,0 +1,51 @@
+/**
+ * Block editor event single v2 reskin
+ *
+ * @since TBD
+ */
+
+/* = Website block view styles
+============================================= */
+.tribe-block__event-website {
+	padding: 20px 0;
+
+	a {
+		background-color: var(--tec-color-link-accent);
```

Comment:
if you're using variables, be sure to add the variables css file as a dependency when defining the asset in php.

Reply:
I've added `tribe-common-full-style` as a dependency because the variables are defined in common. Is this the correct way to do it?

Reply:
`tribe-common-full-style` is the actual common styles, we want the variables. Look for `tec-variables-full` and `tec-variables-skeleton` in common.

Reply:
I've followed the chain and found that the `_variables` file is imported into `common-full` and enqueued as `tribe-common-full-style` together with some other partials. A similar setup exists for `variables-skeleton` in `common-skeleton`. 

Since the variables aren't currently enqueued anywhere on their own without being combined with other style-sheets, should I enqueue them in Common and add them as a dependency in TEC?

Reply:
I've updated these and pulled the latest version of Common as well.